rules.d/75-net-description.rules

```diff
@@ -3,6 +3,8 @@
 ACTION=="remove", GOTO="net_end"
 SUBSYSTEM!="net", GOTO="net_end"
 
+IMPORT{builtin}="hwdb 'net:naming:$env{ID_NET_DRIVER}'"
```

Comment:
i think this should be prepared for future extensions. As mentioned I think we probably want to include the selected net naming scheme in the query sooner or later. Hence maybe encode this similar to how smbios modalias strings are put together, i.e. prefix *and* suffix all matching fields with `:`, and also add in some field id.

hence maybe:

```
IMPORT{builtin}="hwdb 'net:naming:dr$env{ID_NET_DRIVER}:'"
```